src/pages/mydashboard/components/InvitedDashboard/index.tsx

```diff
@@ -0,0 +1,105 @@
+import PageButton from '@/components/Button/PageButton';
+import styles from './InvitedDashboard.module.scss';
+import Image from 'next/image';
+import SearchDashboard from '../SearchDashboard';
+import { Dispatch, SetStateAction } from 'react';
+import HttpClient from '@/apis/httpClient';
+import instance from '@/apis/axios';
+
+type Invitation = {
+  id: number;
+  inviter: { nickname: string; email: string; id: number };
+  teamId: string;
+  dashboard: { title: string; id: number };
+  invitee: { nickname: string; email: string; id: number };
+  inviteAccepted: boolean;
+  createdAt: string;
+  updatedAt: string;
+};
+
+type InvitedDashboardProps = {
+  initialInvitations: Invitation[];
+  invitations: Invitation[];
+  setInvitations: Dispatch<SetStateAction<Invitation[]>>;
+};
+
+function InvitedDashboard({
+  initialInvitations,
+  invitations,
+  setInvitations,
+}: InvitedDashboardProps) {
+  const handleConfirmClick = async (invitationId: number) => {
+    const httpClient = new HttpClient(instance);
+    await httpClient.put(`/invitations/${invitationId}`, {
+      inviteAccepted: true,
+    });
+    const updateData = (await httpClient.get('/invitations')) as {
+      invitations: Invitation[];
+    };
+    setInvitations(updateData.invitations);
+  };
+
+  const handleDenyClick = async (invitationId: number) => {
+    const httpClient = new HttpClient(instance);
+    await httpClient.put(`/invitations/${invitationId}`, {
+      inviteAccepted: false,
+    });
+    const updateData = (await httpClient.get('/invitations')) as {
+      invitations: Invitation[];
+    };
+    setInvitations(updateData.invitations);
+  };
+
+  return (
+    <div className={styles.invitedDashboard}>
+      <strong>초대받은 대시보드</strong>
+      {initialInvitations.length ? (
+        <div>
+          <SearchDashboard
+            initialInvitations={initialInvitations}
+            setInvitations={setInvitations}
+          />
+          <ul>
+            <li>
+              <div className={`${styles.invitedBox} ${styles.title}`}>
+                <p>이름</p>
+                <p>초대자</p>
+                <div className={styles.invitedButton}>수락 여부</div>
+              </div>
+            </li>
+            {invitations.map((invitation) => (
```

Comment:
invitations의 값이 `null`, `undefined`가 올 수 있어서 옵셔널 체이닝을 걸어주면 더 좋을 거 같아요👍
```suggestion
            {invitations?.map((invitation) => (
```